README.md

```diff
@@ -341,6 +341,11 @@ There are certain environment variables used to configure server logging. The de
 
 All the **HTTP requests** get logged by default in the console during development.
 
+#### Mocking captive portal login and logout
+
+This mock is only available when the website is running in development environment. Webpack mode decides the `NODE_ENV` for this mock.
+The endpoints for mocking captive portal login and logout are mentioned [here](https://openwisp-radius.readthedocs.io/en/latest/developer/captive_portal_mock.html).
```

Comment:
avoid links with "here": https://www.w3.org/QA/Tips/noClickHere, use the title of the page as the link text please.

Also avoid long lines, break the lines at about 70 characters (just look at the lines above and try to follow the same line length), this helps us while working on the text in the editor.